src/scss/layout/_footer.scss

```diff
@@ -1,14 +1,15 @@
 .footer {
   color: var(--white);
   background-color: var(--navy-blue);
-  padding: 40px 15px 72px;
+  padding: 48px 0 90px;
```

Comment:
this change is being applied to all the viewports, meaning that the previous styles (`padding: 40px 15px 72px;`) made for mobile will disappear. You need to put this new padding value inside a breakpoint like this: `@include breakpoint(md) { padding: 48px 0 90px; }` And keep the value for mobile outside  of it.